lib/oban_error_reporter.ex

```
@@ -1,5 +1,20 @@
 defmodule ObanErrorReporter do
-  def handle_event([:oban, :job, :exception], measure, %{job: job} = meta, _) do
+  require Logger
+
+  def handle_event(name, measurements, metadata, _) do
```

Just one more thing, shouldn't we have a catch-all clause for the remaining telemetry such as:

```
[:oban, :job, :start]
[:oban, :job, :stop]
```

?

---

Ah nevermind, we attach the handler explicltyl